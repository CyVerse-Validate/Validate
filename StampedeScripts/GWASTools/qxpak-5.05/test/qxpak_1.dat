4 1 3.4 1  16.  33. 4     1
5 1 3.8 2  6.   10. 2.4   2
7 1 3.33 1 7.   13  2.64  2
10 1 3.54 2 8.  16  2.82  2
6 1 3.7 2  19.  12  4.358 1
8 1 3.1 2  14.  14  3.7    1
11 1 2.6 1  8   23  2.8635  2

