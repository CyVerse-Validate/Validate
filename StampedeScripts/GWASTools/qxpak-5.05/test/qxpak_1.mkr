chr1
1   1 1 1 1 1 1 1 1
2   2 2 2 2 2 2 2 2
3   1 2 1 2 1 2 2 1
4   1 2 2 2 2 1 1 2
5   1 1 2 1 1 1 1 1
6   2 2 1 2 2 2 1 2
7   1 1 2 2 1 2 1 2
8   2 2 1 2 1 2 1 2
9   1 2 2 2 1 2 1 2
10  1 2 1 2 2 2 2 2
11  2 1 2 2 1 1 1 1
