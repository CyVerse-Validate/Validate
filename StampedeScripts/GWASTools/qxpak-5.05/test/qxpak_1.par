# Example 1
# Simple F2 cross beteen inbred lines
ML_OPTION
Y
DATAFILE
qxpak_1.dat
OUTFILE
qxpak_1.out
MARKERFILE
qxpak_1.mkr
PEDIGREEFILE
qxpak_1.ped
MARKER_POSITIONS
chr1 0.0 5 10 15
NUMBER_OF_MCMC_ITERATIONS
1000
QTL       
qtl1   fix_ad chr1 all
EFFECT
mean cross 2 
age covariate 3
u0 cross 1 pedigree qxpak_1.ped
TRAIT
t1 5 mean age u0 qtl1
#dummy_trait 6 mean snp2
TEST
qtl1
