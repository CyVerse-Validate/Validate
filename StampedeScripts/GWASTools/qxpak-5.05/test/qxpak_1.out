  
              ***************
              *** QXPAK.5 ***
              ***************
               MPE / IM 2010

 Qxpak v. 5.0   
 Date yy/mm/dd 2010/09/03
 Time hh/mm/ss   16/08/35
 Statistical method:        ML                  
 Parameter file:            qxpak_1.par                             
 Data file:                 qxpak_1.dat                             
 Pedigree file              qxpak_1.ped                             
 Marker file                qxpak_1.mkr                             
 Number of traits:                     1
 Number of inds:                      11
 Number of chrs:                       1
 Number of markers / chr               4
 Number of breeds:                     2
 Marker positions           chr1    0.0    5.0   10.0   15.0
 Number of QTL:                        1
 Number of Epistatic QTL:              0
 Number of effects:                    5
 Number of effects w/o QTL:            3
 Value of missing trait:               0
 QTL scan step (cM):         1.00
 Number of MCMC iters:              1000

SCANNED QTL POSITIONS
     QTL    Type   Imprint   Segment  chr1          
      qtl1  fix_ad     -        -     0.00  15.00

 EFFECTS
 Name          # levels      Type      Randomtype     File
                                      (Nested cov)
 qtl1_a               1   covariable                                 
 qtl1_d               1   covariable                                 
 mean                 1   cross-clas                                 
 age                  1   covariable                                 
 u0                  11   cross-clas   add. animal    qxpak_1.ped    

 INITIAL variances
 Residual
 t1       13.0714
 u0                                      
 t1       10.4571

   1    1.00          -29.764580    19.689

 MODEL           0  specifications
 Trait    Position   Effects
                     qtl1_a    qtl1_d    mean      age       u0        
 t1           5      -         -         +         +         +        

 STATISTICS
 Effect/trait      N    Mean     sd      Min    Max
  mean              7   1.000   0.000   1.000   1.000
  age               7   3.353   0.376   2.600   3.800
  u0                7   7.286   2.373   4.000  11.000
  t1                7  11.143   4.734   6.000  19.000

 Maximum likelihood: 
 2logL = constant +           -29.764580

 RESIDUAL variances
          t1        
 t1       19.6887

 EFFECT variances: u0                                      
          t1        
 t1        8.7543

 EFFECT: h2 in diagonal / correlations off-diagonal u0                                      
          t1        
 t1        0.3078

 SOLUTIONS
 trait/effect level   solution    std. error
  1   3        1        5.0074       16.8395
  1   4        1        1.8516        4.9325
  1   5        1       -0.0000        2.9588
  1   5        2       -0.0000        2.9588
  1   5        3       -0.5083        2.9274
  1   5        4        0.5083        2.9274
  1   5        5       -1.1803        3.2279
  1   5        6        1.3948        3.2180
  1   5        7       -0.8150        3.2030
  1   5        8        0.6353        3.2110
  1   5        9       -0.0000        3.3080
  1   5       10       -0.6957        3.2073
  1   5       11       -0.3557        3.2735

 QTL SCAN
  Positions  df          2logLRatio  Pvalue  QTL_estimates  Residual_Variances
   1    1.00    1         2         9.669555  0.795E-02    -5.529     0.495     6.378
   2    2.00    2         2         9.457670  0.884E-02    -5.914     0.911     6.574
   3    3.00    3         2         8.815469  0.122E-01    -6.224     1.234     7.207
   4    4.00    4         2         7.658194  0.217E-01    -6.256     1.023     8.500
   5    5.00    5         2         6.160747  0.459E-01    -5.878     0.199    10.533
   6    6.00    6         2         5.146585  0.763E-01    -5.390     0.141    12.181
   7    7.00    7         2         4.613140  0.996E-01    -4.961     0.928    13.151
   8    8.00    8         2         4.238216  0.120E+00    -4.516     1.429    13.881
   9    9.00    9         2         3.920062  0.141E+00    -4.041     1.748    14.532
  10   10.00   10         2         3.646684  0.161E+00    -3.572     1.991    15.117
  11   11.00   11         2         3.455089  0.178E+00    -3.164     2.419    15.545
  12   12.00   12         2         3.450055  0.178E+00    -2.696     3.261    15.569
  13   13.00   13         2         3.702294  0.157E+00    -2.003     4.361    15.036
  14   14.00   14         2         4.164331  0.125E+00    -1.249     5.443    13.857
  15   15.00   15         2         5.111580  0.776E-01    -0.576     6.266    12.169

 MODEL           1  specifications
 Trait    Position   Effects
                     qtl1_a    qtl1_d    mean      age       u0        
 t1           5      +         +         +         +         +        

 Likelihood ratio: 
 LR =             9.669555
 Estimated d.f.             2
 Nominal P-value  0.795E-02
 For completeness, nominal P-values for other d.f.
 d.f.  3 P-value  0.216E-01

 QTL positions for LR max    1.00

 RESIDUAL variances
          t1        
 t1        6.3775

 EFFECT variances: u0                                      
          t1        
 t1        0.2531

 EFFECT: h2 in diagonal / correlations off-diagonal u0                                      
          t1        
 t1        0.0382

 SOLUTIONS
 trait/effect level   solution    std. error
  1   1        1       -5.5289        1.3489
  1   2        1        0.4945        2.2915
  1   3        1       -1.9620       10.2830
  1   4        1        3.8232        2.9021
  1   5        1       -0.0000        0.5031
  1   5        2       -0.0000        0.5031
  1   5        3       -0.0447        0.5027
  1   5        4        0.0447        0.5027
  1   5        5       -0.0209        0.5616
  1   5        6        0.0255        0.5617
  1   5        7        0.0231        0.5614
  1   5        8       -0.0229        0.5616
  1   5        9       -0.0000        0.5625
  1   5       10       -0.0850        0.5614
  1   5       11       -0.0092        0.5619
 Total CPU time:           0 h           0 m           0 s
